config/apiconfig.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
We should pack this new `config` folder from the `pack` target alongside the `lib`, `mod` and `native` folders